go.mod

```diff
@@ -158,3 +161,5 @@ replace github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.7.0
 
 // Mock wasmd module removed in experimental build
 replace github.com/CosmWasm/wasmd => ./mocks/wasmd
+
+replace github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8.0
```

Comment:
@aaronc Does this need to also come from our SDK fork? or is that not necessary.

Reply:
OK just double checked the `ics23/go/v0.8.0` tag on SDK upstream. Looks correct to me.